website/src/pages/docs/getting-started/development-workflow.mdx

```diff
@@ -63,14 +72,23 @@ If you are using a monorepo structure, with tools such as [Yarn Workspaces](http
 
 If you need to execute the codegen multiple times, note that you can specify multiple fields for `generates` field, for example:
 
-```yaml {4,7}
-schema: 'server/src/**/*.graphql'
-documents: 'client/src/**/*.graphql'
-generates:
-  client/src/models.ts:
-    - typescript
-    - typescript-operations
-  server/src/models.ts:
-    - typescript
-    - typescript-resolvers
+```ts {7,11}
+import { CodegenConfig } from '@graphql-codegen/cli'
+
+const config: CodegenConfig = {
+  schema: 'server/src/**/*.graphql',
+  documents: 'client/src/**/*.graphql'
+  generates {
```

Comment:
I think so. Because `:` exisits after `generates` in the other file.
https://github.com/dotansimha/graphql-code-generator/blob/a87a2f53fee866d3e444743f81f3104b139553ea/website/src/pages/docs/getting-started/esm-typescript-usage.mdx?plain=1#L34